packages/xo-server/src/api/proxy.js

```diff
@@ -95,9 +95,6 @@ update.params = {
     optional: true,
   },
 }
-update.resolve = {
-  vm: ['vm', 'VM', 'administrate'],
-}
```

Comment:
What about this?

Reply:
`resolde` doesn't support `null` value.

Reply:
I meant: what about the *administrate* permission?

Reply:
The method can only be used by admins. Can an admin don't have a `viewer` permission on a `VM`?

Reply:
Then why did you mention *administrate* before?

Reply:
It's just a habit. We don't need to check permission here because it already checked her https://github.com/vatesfr/xen-orchestra/pull/4722/files#diff-19336457701d48256048daa1d45206f3L76.